clients/java/signalr/src/main/java/HubConnection.java

```diff
@@ -125,6 +145,7 @@ public Subscription on(String target, Action callback) {
             callback.invoke(param1.cast(params[0]), param2.cast(params[1]));
         };
         handlers.put(target, action);
+        logger.log(LogLevel.Trace, "Handler added to target: " + target);
```

Comment:
We should use `String.format` (via the overload I suggested below for Logger) to concatenate strings.